client/signup/steps/website-content/index.tsx

```diff
@@ -232,38 +253,17 @@ export default function WrapperWebsiteContent(
 		  );
 
 	useEffect( () => {
-		async function fetchSelectedPageTitles() {
-			try {
-				const response: {
```

Comment:
Moved this to a react-query hook:
https://github.com/Automattic/wp-calypso/pull/72075/files#diff-c5b55802aa84610eedd9ab19f63a1457b1e04a9257577d1f435cbadea67aa27c

This has a few advantages:
* Removes redundant local state (`pageTitles`, `isLoading`, `isStoreFlow`).
* Separates fetching logic from component logic.
* Mapping of server response to local state is separate.